andela/ah-scorpion
# [#159053981 Feature] Add descriptive validation messages during signup

**What does this PR do?**
Implements the sending of appropriate and descriptive validation messages during sign up

**Description of Task to be completed?**
- Add validators in the serializers for password and email.
- Create a custom password validator function 

**How should this be manually tested?**
- Clone the repository from [here](https://github.com/andela/ah-scorpion) and check out the branch `ft-descriptive-validation-messages-159053981`
- Activate your virtual environment and use the `.env-sample` to set the environment variables.
- Use postman to test these endpoints.
  `POST api/users` for registering
Try this endpoint with the wrong password and email formats and see the error messages sent. 
Also, use valid credentials to sign up and see that there will be no error messages



What are the relevant pivotal tracker stories?
[#159053981](https://www.pivotaltracker.com/n/projects/2184741/stories/159053981)
